envs/safety-gymnasium/safety_gymnasium/tasks/safety_velocity/safety_ant_velocity.py

```diff
@@ -0,0 +1,389 @@
+# Copyright 2022 OmniSafe Team. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+# pylint: disable=all
```

Comment:
Shouldn't we enable the pylint checker?

Reply:
Thanks a lot for carefully reviewing, I have solved it.